yt/utilities/lib/ewah_bool_wrap.pyx

```diff
@@ -901,6 +901,8 @@ cdef class BoolArrayCollection:
             if it_map1 != ewah_coll1[0].end():
                 mi1_ewah1 = dereference(it_map1).second
                 mi1_ewah1.logicalor(mi1_ewah2, ewah_coll3[0][mi1])
+            else:
```

Comment:
@langmm, when you get a chance, could you check this specific change to see if it matches what you think it should?  This was how I got some of the tests to start passing again.